### PR TITLE
Updated styling

### PR DESCRIPTION
**Pull request by tonghx1**

### CODE REVIEW

**Comment by tonghx1** [comment]
I forgot to pull from Tianyu's the first time

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -2,42 +2,37 @@
...
-        <div>
-            <form>
-                <h2 style="color: #0A6644">Add to your Shopping List</h2>
-                <input id="item-name" type="text" name="text-input"  placeholder="  Enter Item.." style="border-radius: 20px; color: #0A6644; width: 300px; height: 30px;">
-                <input type="button" value = "submit" onclick="addToCart()"/>                
-                
+        <div style="text-align: center">
+            <h2 style="color: #0A6644; font-family: 'DM Sans', sans-serif; font: bolder">Add to your Shopping List:</h2>
+            <form action="/form-handler" method="POST">
+                <input name="text-input" placeholder="Enter Item" style="border-radius: 15px; width: 300px; height: 20px; border-color:#0A6644; padding: 5px; padding-left: 15px; font-family: 'DM Sans', sans-serif"></input>
```
Comment (IreneC2020) [inline src/main/webapp/index.html:22]: we need id="item-name" in order for the addToCart() to fetch data

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -2,42 +2,37 @@
...
         <br>
-        <div>
-            <form>
-                <h2 style="color: #0A6644">Add to your Shopping List</h2>
-                <input id="item-name" type="text" name="text-input"  placeholder="  Enter Item.." style="border-radius: 20px; color: #0A6644; width: 300px; height: 30px;">
-                <input type="button" value = "submit" onclick="addToCart()"/>                
-                
+        <div style="text-align: center">
+            <h2 style="color: #0A6644; font-family: 'DM Sans', sans-serif; font: bolder">Add to your Shopping List:</h2>
+            <form action="/form-handler" method="POST">
```
Comment (IreneC2020) [inline src/main/webapp/index.html:21]: we are not using form-handler any more and the method=post now is in addToCart() in script.js

**Review by IreneC2020 — CHANGES_REQUESTED** [review]
please review the comment and make the update so index.html will be able to interact with script.js and return result. 
And I am not seeing the shopping list item will show up after user click submit, will there be other PR on this?
Thanks!

**Pull request closed without merging**